lblod/app-lokaal-mandatenbeheer
# LMB-772 remove tijdsinterval logic

## Description

Remove any PeriodOfTime/Tijdsinterval references and just create it when writing mandatarissen to the LDES. Efficiency might need to be better since now it is just done for every mandataris, might be better to also try to bulk this.
Note: not entirely sure how we should handle data from loket, do we want to include the PeriodOfTime there? And how does it work with the ldes initialisation in that case?

## How to test

Test if you can still create and edit mandatarissen. If you created a mandataris or updated a mandataris, a PeriodOfTime resource should have been written to the ldes with the correct begin and end date of the mandataris. 
Be aware if the mandataris is draft, no periodOfTime will be written to the public ldes.

## Links to other PR's

- https://github.com/lblod/mandataris-service/pull/49
- https://github.com/lblod/frontend-lokaal-mandatenbeheer/pull/337
